requirements-devel.txt

```diff
@@ -21,3 +21,6 @@
 # In applying this license, CERN does not
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+-e git+git://github.com/elastic/elasticsearch-dsl-py.git#egg=elasticsearch-dsl-py
+-e git+git://github.com/elastic/elasticsearch-py.git#egg=elasticsearch-dsl-py
```

Comment:
I think the last one should be `-e git+git://github.com/elastic/elasticsearch-py.git#egg=elasticsearch-py` (no `dsl`) right?

Reply:
yes, you're right! thanks :)